src/main/java/edu/harvard/iq/dataverse/FileDownloadServiceBean.java

```diff
@@ -90,6 +97,7 @@ public void writeGuestbookAndStartBatchDownload(GuestbookResponse guestbookRespo
     }
     
     public void writeGuestbookAndStartBatchDownload(GuestbookResponse guestbookResponse, Boolean doNotSaveGuestbookRecord){
+        boolean original = true; 
```

Comment:
I'm a little confused by how this "original" variable always seems to be true. Maybe I'm missing something.

Reply:
You weren't (missing anything). 
Fixed.